packages/formatter-html/src/formatter.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * @fileoverview A hint formatter that outputs the issues in a HTML file..
+ */
+
+/*
+ * ------------------------------------------------------------------------------
+ * Requirements
+ * ------------------------------------------------------------------------------
+ */
+
+import * as path from 'path';
+
+import * as ejs from 'ejs';
+import * as fs from 'fs-extra';
+
+import { debug as d } from 'hint/dist/src/lib/utils/debug';
+import { IFormatter, Problem, FormatterOptions } from 'hint/dist/src/lib/types';
+import { Category } from 'hint/dist/src/lib/enums/category';
+import * as logger from 'hint/dist/src/lib/utils/logging';
+
+const utils = require('./utils');
```

Comment:
Why can't we do an `import` here?

Reply:
`import` doesn't recognize the file as a module, because has code to be share between node and browser.